contents/blog/product-360.md

```diff
@@ -0,0 +1,124 @@
+---
+date: 2022-01-18
+title: I used to think product people aren't needed. I was wrong
```

Comment:
Title suggestion - _I used to think you don't need product people. I was wrong._ 

(ie. make more active, and add full stop at the end)

Reply:
agreed